lib/common.js

```
@@ -53,7 +53,19 @@ export default [
       "generator-star-spacing": ["error", { before: false, after: true }],
       "arrow-spacing": ["error", { before: true, after: true }],
       "keyword-spacing": ["error", { before: true, after: true }],
-      "indent": [
+      "no-unused-vars": [
+        "error",
+        {
+          caughtErrors: "none",
+        },
+      ],
+      "no-empty": [
```

allow empty catch statements